glossary.md

```diff
@@ -61,4 +61,4 @@ In revision (or version) control systems, a repository is a data structure which
 
 ## Version control
 
-A component of software configuration management, version control, also known as revision control or source control, is the management of changes to documents, computer programs, large web sites, and other collections of information. Changes are usually identified by a number or letter code, termed the "revision number", "revision level", or simply "revision". For example, an initial set of files is "revision 1". When the first change is made, the resulting set is "revision 2", and so on. Each revision is associated with a timestamp and the person making the change. Revisions can be compared, restored, and with some types of files, merged. (source: Fowler, Martin (1 May 2006). "Continuous Integration". martinfowler.com. Retrieved 9 January 2014.)
```

Comment:
can we use this text without attribution copyright-wise?

Reply:
Well, the text is really from Wikipedia, so I don't think it was verbatim copied from the source anyway. If anything, we should acknowledge it.

Reply:
Wikipedia also requires attribution, though, right?

Reply:
Yes, resolving this by rewriting it completely.